operator/v1/types_ingress.go

```diff
@@ -332,6 +340,10 @@ type IngressControllerStatus struct {
 	//     * DNS records have been successfully created.
 	//   - False if any of those conditions are unsatisfied.
 	Conditions []OperatorCondition `json:"conditions,omitempty"`
+
+	// tlsSecurityProfile is the observed TLS connection configuration.
```

Comment:
Could we change "observed" to "effective" (or rephrase as "the TLS connection configuration that is in effect")?

Reply:
Fixed